<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEditDialog</class>
 <widget class="QDialog" name="BookEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>  编辑词库</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix1/rc/actions/edit.png</normaloff>:/new/prefix1/rc/actions/edit.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="2,20,2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,2,2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newRecordButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>添加一条记录</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/new/prefix1/rc/actions/add.png</normaloff>:/new/prefix1/rc/actions/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRecordButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>删除一条记录</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/new/prefix1/rc/actions/remove.png</normaloff>:/new/prefix1/rc/actions/remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="idSelectradioButton">
           <property name="text">
            <string>按 id 查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="wortSelectradioButton">
           <property name="text">
            <string>按单词查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="goTolineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showAllButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>显示全表</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/new/prefix1/rc/actions/sayabout.png</normaloff>:/new/prefix1/rc/actions/sayabout.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ascendingOrderButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>顺序排列</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/new/prefix1/rc/actions/view_sort_ascending.png</normaloff>:/new/prefix1/rc/actions/view_sort_ascending.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="descendingOrderButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>降序排列</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/new/prefix1/rc/actions/view_sort_descending.png</normaloff>:/new/prefix1/rc/actions/view_sort_descending.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
